src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -274,69 +256,107 @@ Workflows will soon be replacing configurations. Learn more about our workflows
   />
 
   To reduce noise, we recommend sending anomalies to a low priority notification channel such as Slack or email.
-  </Collapser>
 
-</CollapserGroup>
-
-### Mute notifications (Slack only) [#mute-notifications]
+  </Collapser>
 
-In Slack, detections coming from specific applications can be muted temporarily or permanently. The entire channel can also be muted temporarily. This is useful in the case of an incident or when the channel should otherwise not be interrupted.
+  <Collapser
+    className="freq-link"
+    id="jira"
+    title="Mute notifications"
+    >
 
-To mute in Slack, select **Mute this app's warnings** or **Mute all warnings**, then select the duration. We will resume sending notifications for any detections once the muting duration has completed.
+  In Slack, detections coming from specific applications can be muted temporarily or permanently. The entire channel can also be muted temporarily. This is useful in the case of an incident or when the channel should otherwise not be interrupted.
 
-Muting an application permanently removes it from the configuration. To add it back in, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Anomaly detection**, and select the configuration to edit.
+  To mute in Slack, select **Mute this app's warnings** or **Mute all warnings**, then select the duration. We will resume sending notifications for any detections once the muting duration has completed.
```

Comment:
I would say what you are muting in Slack after "To mute", just for added clarification

Reply:
"We will resume sending notifications" can probably be shortened to "Notifications will resume" to eliminate the passive voice.